packages/headless/src/app/ssr-engine/types/common.ts

```diff
@@ -122,16 +122,22 @@ export type InferControllersMapFromDefinition<
   TControllers extends ControllerDefinitionsMap<CoreEngine, Controller>,
 > = {[K in keyof TControllers]: InferControllerFromDefinition<TControllers[K]>};
 
-export type InferControllerStaticStateFromDefinition<
-  TDefinition extends ControllerDefinition<CoreEngine, Controller>,
-> = TDefinition extends ControllerDefinition<infer _, infer TController>
-  ? ControllerStaticState<TController['state']>
-  : never;
```

Comment:
Basically, we went from:
* Infer static state map from definitions map = Infer static state from definition

To:
* Infer static state map from controllers map = Infer static state from controller
* Infer static state map from definitions map = Infer controllers map from definitions map = Infer static state map from controllers map